docs/apis/mos/raw_data_pull_api_tokenv2_overview.md

```diff
@@ -0,0 +1,91 @@
+| [Clicks (Protect360 fraud)](https://dev.appsflyer.com/hc/reference/get_app-id-blocked-clicks-report-v5)                     | `/{app-id}/blocked_clicks_report/v5`            | `GET`      | Records clicks performed by users blocked by Protect360. Data freshness: Daily                                                                                                        |
+| [Blocked install postbacks](https://dev.appsflyer.com/hc/reference/get_app-id-blocked-install-postbacks-v5)                 | `/{app-id}/blocked_install_postbacks/v5`        | `GET`      | Records install postbacks that were blocked due to being identified as fraudulent. Data updated in real time.                                                                         |
+| **Postbacks**                                                                                                               |                                                 |            |                                                                                                                                                                                       |
+| [Install postbacks](https://dev.appsflyer.com/hc/reference/get_app-id-postbacks-v5)                                         | `/{app-id}/postbacks/v5`                        | `GET`      | Records install events generated when a user opens the app for the first time. Data freshness: Daily                                                                                  |
+| [In-app event postbacks](https://dev.appsflyer.com/hc/reference/get_app-id-in-app-events-postbacks-v5)                      | `/{app-id}/in-app-events-postbacks/v5`          | `GET`      | Records in-app event postbacks sent to the media source. Data freshness: Daily                                                                                                        |
+| [Retargeting in-app event postbacks](https://dev.appsflyer.com/hc/reference/get_app-id-retarget-in-app-events-postbacks-v5) | `/{app-id}/retarget_in_app_events_postbacks/v5` | `GET`      | Records in-app events users performed during the re-engagement window. Data freshness: Real-time                                                                                      |
+| [Retargeting conversions postbacks](https://dev.appsflyer.com/hc/reference/get_app-id-retarget-install-postbacks-v5)        | `/{app-id}/retarget_install_postbacks/v5`       | `GET`      | Records retargeting conversion postbacks sent to the media source. Data freshness: Real-time                                                                                          |
+
+## Path parameters
+The path parameters are identical for all raw data pull APIs. 
+
+| Parameter | Data Type | Description    | Example    |
+| --------- | --------- | -------------- | ---------- |
+| `app_id*` | string    | Application ID | `id121244` |
+
+## Query parameters
+The query parameters are identical for all raw data pull APIs
```

Comment:
This is not true.

For example:
`blocked_installs_report/v5` has  `from_install_time` and `to_install_time` and  `/ad-revenue-raw-retarget/v5` does not.

I suggest you review again the API in the OAS file https://github.com/AppsFlyerKnowledge/devhub/blob/main/docs/apis/mos/raw_data_pull_api_tokenv2.yaml and see the differences in a clearer way